### PR TITLE
Add Korean 300M community model to README

### LINKED ISSUE #261: Korean pocket-tts model
Thanks for updating training code!

I could train korean model, and seems working.

- **100M student:** https://huggingface.co/seastar105/pocket-tts-korean-100m
- **300M teacher:** https://huggingface.co/seastar105/pocket-tts-korean-300m

Student models have quite poor performance while teacher is quite usable one.

## Usage

```bash
  uvx pocket-tts generate \
      --config hf://seastar105/pocket-tts-korean-100m/korean.yaml \
      --voice ./voice_prompt.wav \
      --text "안녕하세요. 한국어 음성 합성 모델입니다."
```

 ## Training

I trained the models using this Korean dataset:
https://huggingface.co/datasets/seastar105/emilia-yodas-ko-filtered-pocket-tts

The dataset contains around 2k hours(900k samples) from yodas2-ko. And i used qwen3-forced-aligner to get alignment. 

When I trained model from scratch, it collapsed fast at the beginning of the train(~1,2k steps). So I warm-started the 300M teacher from English 24-layer checkpoint. 

Samely, I initialized the 100M student from the English 6-layer checkpoint and depth-distilled it from the Korean teacher. It has much advantage.

Both models are trained with 50k steps with 4 x RTX 5090 cards, code is [here](https://github.com/seastar105/pocket-tts-korean-training/tree/main/training)

I archived all intermediate checkpoints here:
- https://huggingface.co/seastar105/pocket-tts-checkpoints

## Evaluation

I evaluated all 25 EMA checkpoints from each training run on all 500 Korean zero-shot items from yuekai/CV3-Eval `zero_shot_ko`.

CER metric gets low very fast that flats while UTMOS kepp going high during training teacher model, while student training UTMOS,CER getting worse

Image

Image

## NOTE

These models are very poor at english, numbers reading, while acoustic quality and speaker sim is quite good. I think it's reasonable as dataset size is not large and training steps are small. 

Poor student model performance may get better if model starts from teacher-init and training longer.

### PR DESCRIPTION
**Pull request by seastar105**
## Summary

- Add the community-trained Korean 300M Pocket TTS model to the README
- Include a pinned Hugging Face config and Korean generation example
- Document only the 300M teacher model

close #261

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -303,6 +303,14 @@ uvx pocket-tts generate \
   --text "आज का दिन बहुत अच्छा है"
 ```
 
+- [Pocket TTS Korean 300M](https://huggingface.co/seastar105/pocket-tts-korean-300m) by [@seastar105](https://huggingface.co/seastar105) (community research release):
+```bash
+uvx pocket-tts generate \
+  --config hf://seastar105/pocket-tts-korean-300m/korean.yaml@df328c817a02866f20a6f74e5183e0a1fc6f6435 \
+  --voice ./voice_prompt.wav \
+  --text "안녕하세요. 한국어 음성 합성 모델입니다."
```
Comment (gabrieldemarmiesse-ai-agent) [inline README.md:310]:
```suggestion
  --config hf://seastar105/pocket-tts-korean-300m/korean.yaml@df328c817a02866f20a6f74e5183e0a1fc6f6435 \
  --text "안녕하세요. 한국어 음성 합성 모델입니다."
```

To make it even easier to try out we can remove the `--voice` no? It's going to use the "alba" wav file, which is an english voice but that should work for voice cloning. Can you try it out?

**Comment by seastar105** [comment]
i found use no `--voice` works, and applied it

**Comment by gabrieldemarmiesse** [comment]
Awesome, many thanks for this TTS!
